orangewidget/utils/itemmodels.py

```diff
@@ -496,11 +498,48 @@ def remove(self, val):
         del self[self._table.index(val)]
 
 
+class SeparatorItem:
+    pass
+
+
+class LabelledSeparator(SeparatorItem):
+    def __init__(self, label=None):
+        self.label = label
+
+
+class SeparatedListDelegate(QStyledItemDelegate):
+    def paint(self, painter, option, index):
+        # type: (QPainter, QStyleOptionViewItem, QModelIndex) -> None
+        super().paint(painter, option, index)
+        data = index.data(Qt.EditRole)
+        if not isinstance(data, LabelledSeparator):
+            return
+
+        painter.save()
+        palette = option.palette  # type: QPalette
+        rect = option.rect  # type: QRect
+        if data.label:
+            y = int(rect.bottom() - 0.1 * rect.height())
+            brush = palette.brush(QPalette.Active, QPalette.WindowText)
+            font = painter.font()
+            font.setPointSize(font.pointSize() * 0.9)
```

Comment:
Crashes for me (Python 3.10, Qt 6), because `setPointSize` expects an int. Use `setPointSizeF` (and `pointSizeF`).

This also means that this delegate's paint is never tested.

Reply:
Thanks. I fixed that and added a test. It doesn't tests arguments of all calls to painter, but at least test all paths.